compiler/api/compiler.py

```diff
@@ -447,7 +447,7 @@ def start(format: bool = False):
                     )
 
                     read_types += "\n        "
-                    read_types += "{} = TLObject.read(b{}) if flags & (1 << {}) else []\n        ".format(
+                    read_types += "{} = TLObject.read(b{}) if flags & (1 << {}) else None\n        ".format(
```

Comment:
What was this breaking exactly? I'm not sure I understood what you said about this change.

Reply:
TL;DR

```py3
if self.logout_tokens is not None:
    b.write(Vector(self.logout_tokens, Bytes))
```
Breaks

Reply:
Makes sense. I think it's better to keep your first proposal and change `[]` to `None`.